src/renderer/hooks/useSatStackStatus.js

```diff
@@ -3,10 +3,20 @@ import { useMemo } from "react";
 import { useObservable } from "@ledgerhq/live-common/lib/observable";
 import type { SatStackStatus } from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
 import { command } from "~/renderer/commands";
+import { filter, timeoutWith } from "rxjs/operators";
+import { of } from "rxjs";
 
 let lastState;
+const INITIAL_DISCONNECT_TIMEOUT = 10000;
 const useSatStackStatus = (): ?SatStackStatus => {
-  const observable = useMemo(() => command("getSatStackStatus")(), []);
+  const observable = useMemo(
+    () =>
+      command("getSatStackStatus")().pipe(
+        filter(({ type }) => type !== "satstack-disconnected"),
+        timeoutWith(INITIAL_DISCONNECT_TIMEOUT, of({ type: "satstack-disconnected" })),
+      ),
+    [],
+  );
```

Comment:
I'm not sure to understand what this fixes, typically to me it doesn't fix https://ledgerhq.atlassian.net/browse/LL-4021 but the other code does (the code changed in StepSatStack)

To me we really don't want to timeout, specifically not if you watch the Settings tab, it should runs forever, not stops after 10s 🤔

Reply:
unless I did it wrong, the command is triggered every second, what this does is to delay showing the disconnect one until after 10 seconds, any other status will work 🤷. Without this it will show the disconnected status directly